src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.css

```diff
@@ -25,6 +25,7 @@
 }
 
 .domainPotBalance, .networkFee {
+  margin-top: -16px;
```

Comment:
Please leave a comment explaining the use of negative margins

Reply:
Not saying it needs to be changed, although a more robust alternative approach to this could be to use `position: absolute`, with right and bottom positioning and `position: relative` on the fields.

Reply:
> Please leave a comment explaining the use of negative margins

Because the `inputStatus` css component has visibility hidden and this is making the error message taking up space so my thought was to use the negative margin to get around that issue.  

The other alternative I was thinking was adding another message field to this component but it seemed that this would be too involved for this change? What do you think?

Reply:
> Not saying it needs to be changed, although a more robust alternative approach to this could be to use `position: absolute`, with right and bottom positioning and `position: relative` on the fields.

I could give it a try :) thanks for the suggestion.

Reply:
> > Please leave a comment explaining the use of negative margins
> 
> Because the `inputStatus` css component has visibility hidden and this is making the error message taking up space so my thought was to use the negative margin to get around that issue.
> 
> The other alternative I was thinking was adding another message field to this component but it seemed that this would be too involved for this change? What do you think?

Lol. Please leave a comment, in the css file :)

Reply:
Hey @arrenv I ended up using relative positioning and setting the bottom. It didn't seem that the absolute positioning was necessary but can you please take a look at it again so I can merge this? Thank you!